index.html

```diff
@@ -552,30 +528,50 @@ <h4>Creators</h4>
+							</ul>
+						</li>
+					</ul>
 
-					<p>The language specified in the <a>Web Publication's</a>
-						<abbr title="information set"><a>infoset</a></abbr> identifies the natural language of its
-						properties.</p>
+					<p>
+						Furthermore, the <a>Web Publication's</a> <abbr title="information set"><a>infoset</a></abbr> MAY contain global values for the language and the base direction using a [[!bcp47]] tag, and the values of <code>ltr</code>/<code>rtl</code>/<code>auto</code> as above, respectively. These are used as defaults for textual properties in the <abbr title="information set"><a>infoset</a></abbr>, unless overwritten by the property itself.
+					</p>
 
-					<p>When specified, the language MUST be a tag that conforms to [[!bcp47]].</p>
+					<p class="note">
+						These features make it possible to add the title of a publication in different languages, or repeat the creators’ name using different scripts.
+					</p>
 
-					<p class="note">This language is not used in the processing or rendering of the Web Publication, and is
-							<strong>not</strong> a replacement for identifying the language of each resource as defined by
-						its format.</p>
+					<p class="note">
+						This default language/direction pair is not used in the processing or rendering of the Web Publication, and is
+						<strong>not</strong> a replacement for identifying the language of each resource as defined by
+						its format. These information items have an effect on the metadata expressed in the <a>Web Publication's</a> <abbr title="information set"><a>infoset</a></abbr> only.
+					</p>
 
-					<p>If a user agent requires the language and one is not available in the <abbr title="information set"
-							>infoset</abbr>, it MAY attempt to determine the language. This specification does not mandate
-						how such a language tag is created. The user agent might:</p>
+					<p>
+						If a user agent requires the language and one is not available in the <abbr title="information set">infoset</abbr>,
+						nor is it specified for a specific property, it MAY attempt to determine the language. This specification 
+						does not mandate how such a language tag is created. The user agent might:
+					</p>
 
 					<ul>
 						<li>use the <a>non-empty</a> language declaration of the manifest;</li>
-						<li>use the first non-empty language declaration found in a resource in the <a>default reading
-								order</a>;</li>
+						<li>use the first non-empty language declaration found in a resource in the <a>default reading order</a>;</li>
 						<li>calculate the language using its own algorithm.</li>
 					</ul>
 
-					<p>If a language tag cannot be determined, the value "<code>und</code>" (undetermined) MUST be used.</p>
+					<p>If a language tag cannot be determined, the value "<code>und</code>" (undetermined) MUST be used. If the base direction cannot be determined, the value <code>ltr</code> MUST be assumed.</p>
```

Comment:
`auto` might be a better default.

Reply:
oops... of course. Will take care of this.